app/src/NativeBase/Components/Checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file CheckBox component for the STA Volunteer App
+ */
+
+import { Pressable } from 'native-base'
+import React from 'react'
+import { Ionicons } from '@expo/vector-icons'
```

Comment:
Was there a reason not to use a Material Icons icon here?  The one you've used looks fine, but just for consistency with the rest of the project https://github.com/Scottish-Tech-Army/Volunteer-app/blob/develop/APP_DEVELOPMENT.md#icons

Reply:
Used the above as came with expo-checbox. I didn't appreciate we used MaterialIcons but i can look into swapping this out.